site/pages/servicecomplexe/page2-fr.hbs

```diff
@@ -46,14 +46,14 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Précedent</span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/index-fr.html" class="btn btn-default"><span></span>Précedent<span class="wb-inv"> return to the previous page</span></a></div></li>
```

Comment:
No need for absolute links as the relative link value is in the `{{link}}` variable.